utils/upgrade.js

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env node
+
+/**
+ * Copyright Zendesk, Inc.
+ *
+ * Use of this source code is governed under the Apache License, Version 2.0
+ * found at http://www.apache.org/licenses/LICENSE-2.0.
+ */
+
+const path = require('path');
+const { spawnSync } = require('child_process');
+
+const pkg = require('../package.json');
+
+(({ submodule, containers, components }) => {
+  if (submodule) {
+    spawnSync('git', ['pull'], { cwd: path.resolve('react-components'), stdio: 'inherit' });
```

Comment:
Along with `git pull`, the submodule should be set to the latest tag (rather than `head`) via `git checkout v8.60.0` for example.